### PR TITLE
feat: export plugin config types

### PR DESCRIPTION
**Pull request by Leomotors**
I would want to export the plugin's config type so user can also type their prettier config, they will look something like this.

```js
// @ts-check

/** @type {import("@trivago/prettier-plugin-sort-imports").PrettierConfig} */
const config = {
    bracketSpacing: true,
    tabWidth: 4,
    useTabs: false,
    singleQuote: false,
    semi: true,
    printWidth: 80,
    plugins: ["@trivago/prettier-plugin-sort-imports"],
    importOrderCaseInsensitive: true,
    importOrderSeparation: true,
    importOrderSortSpecifiers: true,
    importOrderParserPlugins: ["typescript", "decorators-legacy", "jsx"],
};

module.exports = config;
```

This will help strongly type all parts of their apps, increasing code quality. Please consider this PR 🙏

If you have any concerns like convention, naming, feel free to request changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,4 +54,4 @@
   "peerDependencies": {
     "prettier": "2.x"
   }
-}
+}
```
Comment (byara) [inline package.json:57]: Would you please add a new line at the end of this file or run prettier here?

**Review by byara — COMMENTED** [review]
Thank you for your contribution 🚀 
I left a comment, but in general looks very good 👍

**Review by byara — APPROVED** [review]
Thanks for the PR 🎉

**Review by ayusharma — APPROVED** [review]
Looks good to me. 🚀 Thank you so much for the contribution.

**Pull request merged**